dips/dip-10.md

```diff
@@ -65,21 +76,21 @@ struct DiemIdDomain {
 }
 ```
 * Fields definition:
-   * `domain` - name of a Diem ID domain 
+   * `domain` - name of a Pay Address domain 
 * The `DiemIdDomains` resource can only be published into an account with `Role == PARENT_VASP_ROLE_ID`.
-* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Diem ID Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
+* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Pay Address Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
```

Comment:
I dunno if w need to say why more than one would be beneficial... just simply stating that a VASP can potentially have 1 or more PayAddressDomains